lib/formats/bs_format_rislive.c

```diff
@@ -610,45 +627,18 @@ int bs_format_rislive_get_next_elem(bgpstream_format_t *format,
     break;
   case RISLIVE_MSG_TYPE_STATUS:
     RDATA->elem->type = BGPSTREAM_ELEM_TYPE_PEERSTATE;
-    switch (RDATA->status_msg_state) {
-    case 0:
-      RDATA->elem->old_state = BGPSTREAM_ELEM_PEERSTATE_UNKNOWN;
-      RDATA->elem->new_state = BGPSTREAM_ELEM_PEERSTATE_IDLE;
-      break;
-    case 1:
-      RDATA->elem->old_state = BGPSTREAM_ELEM_PEERSTATE_UNKNOWN;
-      RDATA->elem->new_state = BGPSTREAM_ELEM_PEERSTATE_ESTABLISHED;
-      break;
-    default:
-      RDATA->elem->old_state = BGPSTREAM_ELEM_PEERSTATE_UNKNOWN;
-      RDATA->elem->new_state = BGPSTREAM_ELEM_PEERSTATE_UNKNOWN;
-      bgpstream_log(BGPSTREAM_LOG_WARN, "unsupported status type, %d",
-                    RDATA->status_msg_state);
-      break;
-    }
+    RDATA->elem->old_state = BGPSTREAM_ELEM_PEERSTATE_UNKNOWN;
+    RDATA->elem->new_state = RDATA->status_msg_state;
     RDATA->end_of_elems = 1;
     rc = 1;
     break;
   case RISLIVE_MSG_TYPE_OPEN:
+    bgpstream_log(BGPSTREAM_LOG_WARN, "RIS Live OPEN support coming soon...");
```

Comment:
how soon? before v2 release?
if not, i think we should remove this warning and we'll just have to document that we don't support it.

Reply:
i think it requires quite some efforts and i vote to put this into v2.1 milestone.